proto/v1/idp_service.proto

```diff
@@ -68,7 +69,10 @@ service IdentityProviderService {
 }
 
 message GetIdentityProviderRequest {
-  string name = 1 [(google.api.field_behavior) = REQUIRED];
+  string name = 1 [
+    (google.api.field_behavior) = REQUIRED,
+    (google.api.resource_reference) = {type: "bytebase.com/IDP"}
```

Comment:
`IdP` or `IdentityProvider`?

<img width="965" alt="image" src="https://github.com/user-attachments/assets/a4cd4735-4286-42e8-80f6-547e49ea0390">

https://cloud.google.com/iam/docs/workload-identity-federation-with-other-providers

Reply:
We have the resource "idps/{idp}" so I'm inclining to IdP.